http-api-samples/http-api-owner-sample/src/main/java/org/fidoalliance/fdo/api/OwnerCustomerServlet.java

```diff
@@ -69,7 +69,11 @@ protected void doDelete(HttpServletRequest req, HttpServletResponse resp)
 
     try {
       DataSource ds = (DataSource) getServletContext().getAttribute("datasource");
-      new OwnerDbManager().removeCustomer(ds, customerId);
+      int rowsUpdated = new OwnerDbManager().removeCustomer(ds, customerId);
+      if (rowsUpdated == 0) {
+        resp.setStatus(HttpServletResponse.SC_NOT_FOUND);
```

Comment:
Add a logger.warn() statement here => 

logger.warn("Request failed because of invalid customer ID");